appveyor.yml

```diff
@@ -11,7 +11,7 @@ environment:
   matrix:
       - PYTHON_VERSION: "3.7"
         CONDA_DEPENDENCIES: "numpy scipy setuptools pytest coverage pytest-cov pytest-sugar pytest-faulthandler cython coveralls freetype-py numpydoc meshio"
-        PIP_DEPENDENCIES: ""
+        PIP_DEPENDENCIES: "pyopengltk"
```

Comment:
The testing on appveyor is pretty limited. I would just remove this and not test on Windows. It is difficult to fully test (accurately) the visualization stuff on Appveyor (or Windows CI). At least it has been in the past.

Reply:
I get some the same errors as [shown on Appveyor](https://ci.appveyor.com/project/vispy/vispy/builds/35423539) locally on Windows, but the tests pass as I said previously. I'll remove the dependencies for Appveyor than though.

Reply:
Yeah I don't think we have the proper GL stuff setup on Appveyor. Errors like:

```
AttributeError: 'LooseVersion' object has no attribute 'version'
```

although not very helpful (there's a bug report for that) usually indicate OpenGL isn't properly installed or available in a way that vispy can find it.